samples/StaticFilesAuth/StaticFilesAuth.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
     <UserSecretsId>aspnet-StaticFilesAuth-AFE2BD9D-1575-4C3E-BE32-3F15C5BC9947</UserSecretsId>
+    <NoWarn>NU1605</NoWarn>
```

Comment:
I recommend putting this in Directory.Build.props instead of once in each project, and changing to
```
<NoWarn>$(NoWarn);NU1605</NoWarn>
```

Easier to clean up later.

Reply:
Makes sense. Will do